aten/src/THS/generic/THSTensor.cpp

```diff
@@ -57,6 +57,12 @@ THTensor *THSTensor_(newValues)(const THSTensor *self) {
   return THTensor_(newNarrow)(self->values, 0, 0, self->nnz);
 }
 
+THLongTensor *THSTensor_(newCsr)(const THSTensor *self) {
+  if (self->csr == NULL) {
+    return self->csr;
+  }
+  return THLongTensor_newNarrow(self->csr, 0, 0, self->size[0] + 1);
+}
```

Comment:
Can we call this differently? Perhaps `getCSR`? Having functions called `newCsr` and `newCSR`, and doing different things is really confusing

Reply:
`newCSR` is now named `newWithCSR`, and `newCsr` is now named `newCSR`.